packages/lib/createEmailExclusionRegex.test.ts

```diff
@@ -0,0 +1,59 @@
+import { describe, it, expect } from "vitest";
+
+import { createEmailExclusionRegex } from "./createEmailExlusionRegex";
+
+describe("createExclusionRegex", () => {
+  it("should create regex for full email match", () => {
+    const exclusionString = "spammer@cal.com";
+    const regex = createEmailExclusionRegex(exclusionString);
+
+    expect(regex.test("spammer@cal.com")).toBe(true);
+    expect(regex.test("user@cal.com")).toBe(false);
+  });
+
+  it("should create regex for domain match", () => {
+    const exclusionString = "gmail.com";
+    const regex = createEmailExclusionRegex(exclusionString);
+
+    expect(regex.test("user@gmail.com")).toBe(true);
+    expect(regex.test("user@yahoo.com")).toBe(false);
+    expect(regex.test("@gmail.com")).toBe(true);
+  });
```

Comment:
We don't really need to test the regex, but we do need to test the functionality. Pls add it

Reply:
@Amit91848 You're almost there! Please refer to the link below to add test functionality. I previously worked partially on a similar feature and created a test file, which you might find helpful. You can refer to it and add unit test cases tailored to your current use case.
packages/features/form-builder/lib/formbuilder/test/FormBuilder.test.tsx